gnomad_qc/v4/annotations/generate_freq.py

```diff
@@ -367,6 +364,22 @@ def correct_age_hists(ht: hl.Table) -> hl.Table:
     )
 
 
+# TODO: I think `qual_hist_expr` in gnomad_methods should just be modified to either
+#  have an option to return them like this or to return them like this by default and
+#  just have a breaking change.
+#    if adj_expr is not None:
+#        adj_qual_hists = {
+#            qual_hist_name: hl.agg.filter(adj_expr, qual_hist_expr)
+#            for qual_hist_name, qual_hist_expr in qual_hists.items()
+#        }
+#        if split_adj_and_raw:
+#            return hl.struct(
+#                raw_qual_hists=hl.struct(**qual_hists),
+#                qual_hists=hl.struct(**adj_qual_hists)
+#            )
+#        else:
+#            qual_hists.update({f"{k}_adj": v for k, v in adj_qual_hists.items()})
+#    return hl.struct(**qual_hists)
 def compute_qual_hists(mt: hl.MatrixTable) -> hl.MatrixTable:
```

Comment:
I think `qual_hist_expr` in gnomad_methods should just be modified to either have an option to return them like this or to return them like this by default and just have a breaking change.

Modification could just be to add `split_adj_and_raw` parameter and then change end of function to something like this:
```
    if adj_expr is not None:
        adj_qual_hists = {
            qual_hist_name: hl.agg.filter(adj_expr, qual_hist_expr)
            for qual_hist_name, qual_hist_expr in qual_hists.items()
        }
        if split_adj_and_raw:
            return hl.struct(
                raw_qual_hists=hl.struct(**qual_hists),
                qual_hists=hl.struct(**adj_qual_hists)
            )
        else:
            qual_hists.update({f"{k}_adj": v for k, v in adj_qual_hists.items()})

    return hl.struct(**qual_hists)
```